src/main/java/com/alkemy/ong/exception/NotFoundExceptions.java

```diff
@@ -0,0 +1,8 @@
+package com.alkemy.ong.exception;
+
+
+public class NotFoundExceptions extends RuntimeException{
```

Comment:
Esta buenisimo que hayas creado esta exception! Te consulto, se esta devolviendo el mensaje correspondiente en caso de error? Cuando es un not found se debe devolver un 404, chequeate eso y si no se devuelve habria que agregarlo

Reply:
Si devuelve 404.